### ISSUE TITLE
Error with 0.2.4. Rollback to 0.2.3 and it works again...

### ISSUE THREAD

**Issue by Dominik-1980** [samemory/homebridge-eufy-security#57]
**Describe The Bug:**

Update to 0.2.4 and I got an error when I restart homebridge. 
I have two cameras and both are showing the error message for the motion sensor...
Downgrade to 0.2.3 and everything is working again...

**To Reproduce:**

Just update the plugin.

**Logs:**

```
(node:13132) UnhandledPromiseRejectionWarning: Error: This callback function has already been called by someone else; it can only be called one time.
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/util/once.ts:6:13
    at SecurityCameraAccessory.handleMotionOnGet (/homebridge/node_modules/homebridge-eufy-security/src/securityCameraAccessory.ts:239:5)
    at On.emit (events.js:387:35)
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1508:14
    at new Promise (<anonymous>)
    at On.<anonymous> (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1506:12)
    at step (/usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:143:27)
    at Object.next (/usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:124:57)
    at /usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:117:75
    at new Promise (<anonymous>)
(node:13132) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:13132) UnhandledPromiseRejectionWarning: Error: This callback function has already been called by someone else; it can only be called one time.
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/util/once.ts:6:13
    at SecurityCameraAccessory.handleMotionOnGet (/homebridge/node_modules/homebridge-eufy-security/src/securityCameraAccessory.ts:239:5)
    at On.emit (events.js:387:35)
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1508:14
    at new Promise (<anonymous>)
    at On.<anonymous> (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1506:12)
    at step (/usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:143:27)
    at Object.next (/usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:124:57)
    at /usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:117:75
    at new Promise (<anonymous>)
(node:13132) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
```

**Plugin Config:**

```json
{
            "username": "xxx@gmail.com",
            "password": "xxx",
            "hkHome": 1,
            "hkAway": 0,
            "hkNight": 3,
            "hkOff": 63,
            "platform": "EufySecurity"
        }
```

**Environment:**

* **Plugin Version**: 0.2.4
* **Homebridge Version**: 1.3.4
* **Node.js Version**: 14.17.1
* **NPM Version**: 6.14.13
* **Operating System**: Raspbian / Docker

**Comment by lenoxys** [comment]
Yeah #54 should fix the issue but no new version has been release. @samemory can you create the 0.2.5 ?

**Comment by Dominik-1980** [comment]
Okay. Thank you.
So I will wait till 0.2.5

**Comment by njuart** [comment]
It looks similar to issue I've reported before [56](https://github.com/samemory/homebridge-eufy-security/issues/56). I suppose 0.2.5 version should fix it right? Thx

**Comment by lenoxys** [comment]
it should be fixed on 0.2.5
